components/compositing/Cargo.toml

```diff
@@ -25,6 +25,9 @@ path = "../script_traits"
 [dependencies.style_traits]
 path = "../style_traits"
 
+[dependencies.style]
+path = "../style"
```

Comment:
This looks like a rebase error to me; there are no new users of the style crate in these changes.